tests/api/testdata/courses/assignments/proxy/courses_assignments_submissions_proxy_regrade_base.json

```diff
@@ -0,0 +1,32 @@
+{
+    "module": "autograder.api.courses.assignments.submissions.proxy.regrade",
+    "read-write": true,
+    "arguments": {
+        "dry-run": false,
+        "overwrite-records": false,
+        "wait-for-completion": false
+    },
+    "output-modifier": "clean_output_timestamps_and_submission_ids",
+    "output": {
+		"options": {
+			"dry-run": false,
+			"overwrite-records": false,
+			"wait-for-completion": false,
+			"target-users": [
+				"*"
+			],
+			"regrade-after": 1234567890123
+		},
+		"regrade-after": 1234567890123,
```

Comment:
I'm not a fan of this duplicate option.

I understand that the first one is a pointer and the second one is not,
but the first one is now (since a few PR passes ago) also non-nil.